.github/workflows/main.yml

```diff
@@ -160,7 +160,7 @@ jobs:
       - run: |
           NEW_JAR=$(realpath $(find . -name photonvision\*.jar))
           sudo apt install unzip zip
-          curl -sk https://api.github.com/repos/photonvision/photon-pi-gen/releases/latest | grep "browser_download_url.*zip" | cut -d : -f 2,3 | tr -d '"' | wget -qi -
+          curl -sk https://api.github.com/repos/photonvision/photon-pi-gen/releases/tags/v2021.1.4 | grep "browser_download_url.*zip" | cut -d : -f 2,3 | tr -d '"' | wget -qi -
```

Comment:
Was this meant to be committed?

Reply:
Yes - per https://discord.com/channels/725836368059826228/725846807019782174/915598652934881322  and the conversation before it - goal is to allow photon to choose explicitly when it move up and down in pigen versions.

Also happens to resolve the weirdness of github calling the first tag on a particular node "Latest"